src/utils/Decimals.ts

```diff
@@ -0,0 +1,3 @@
+export default function reduceDecimals(number) {
+  return String(parseFloat(number).toFixed(18))
```

Comment:
This is not ok, please check what we did in https://github.com/oceanprotocol/market/pull/586 . We should be consistent  all over the place. Of course , improvements are welcomed.